frontend/src/reducer/tlmManager.js

```diff
@@ -1,9 +1,11 @@
-import { SET, TLM_MANAGER } from '../action-creator.sync.js'
+import { SET, TLM_MANAGER, TLM_MANAGER_STATUS } from '../action-creator.sync.js'
 
-export default function tlmManager (state = null, action) {
+export default function tlm (state = {}, action) {
```

Comment:
If you rename the reducer, also rename its filename please

Reply:
Also, now you have multiple properties, its better to add them to its default value
```jsx
const defaultTlm = {
  manager: null,
  status: LIVE_MESSAGE_STATUS.CLOSED
}
export default function tlm (state = defaultTlm, action) {
```